sonic_data_client/mixed_db_client.go

```diff
@@ -1110,7 +1113,7 @@ func (c *MixedDbClient) handleTableData(tblPaths []tablePath) error {
 }
 
 /* Populate the JsonPatch corresponding each GNMI operation. */
-func (c *MixedDbClient) ConvertToJsonPatch(prefix *gnmipb.Path, path *gnmipb.Path, t *gnmipb.TypedValue, output *string) error {
+func (c *MixedDbClient) ConvertToJsonPatch(prefix *gnmipb.Path, path *gnmipb.Path, t *gnmipb.TypedValue, operation string, output *string) error {
```

Comment:
>operation [](http://example.com/codeflow?start=105&length=9)

Could you PR title or PR description for this new feature?

Reply:
updated.